superset/commands/exceptions.py

```diff
@@ -28,7 +28,7 @@ class CommandException(SupersetException):
     def __repr__(self) -> str:
         if self._exception:
             return repr(self._exception)
```

Comment:
Should this line also be updated?

Reply:
This isn't recursive, as it is calling `repr` on a separate object